packages/sdk-react-native/index.ts

```diff
@@ -134,9 +126,9 @@ class iOSPasskeys implements CredentialSigner<Fido2Assertion>, CredentialStore<F
         name: challenge.user.name,
       },
       attestation: challenge.attestation,
-      excludeCredentials: challenge.excludeCredentials.map((cred) => ({
-        id: cred.id,
-        type: cred.type,
+      excludeCredentials: challenge.excludeCredentials.map(({ id, type }) => ({
+        id: b64UrlSafeToStandard(id),
```

Comment:
Have we been handling this incorrectly?

Reply:
yes, but it's only hit when you create an additional credential. on registration, there's no `excludeCredentials`. I don't think anyone's hit this problem yet since the react native integration is fairly new.